src/test/java/nl/talsmasoftware/umldoclet/html/HtmlFileTest.java

```diff
@@ -44,9 +44,9 @@ public class HtmlFileTest {
 
     @BeforeAll
     public static void createTempdir() throws IOException {
-        tempdir = File.createTempFile("HtmlFile", "-test");
-        assertThat("Delete tempfile", tempdir.delete(), is(true));
-        assertThat("Create temporary directory", tempdir.mkdirs(), is(true));
+        tempdir = Files.createTempDirectory("HtmlFile" + "-test").toFile();
+        assertThat("Delete tempfile", true, is(true));
+        assertThat("Create temporary directory", true, is(true));
```

Comment:
There was no security vulnerability in the first place due to these assertions.
Now that you've rewritten the code to use `createTempDirectory` instead, please also remove these now-obsolete assertions.

Reply:
The vulnerability is still present, but different. In this case, it's temporary directory information disclosure, not hijacking. Regardless, I see your point.